public/app/features/explore/LogsContainer.tsx

```diff
@@ -133,37 +122,37 @@ class LogsContainer extends PureComponent<LogsContainerProps> {
           </Collapse>
         </LogsCrossFadeTransition>
         <LogsCrossFadeTransition visible={!isLive}>
-          <Collapse label="Logs" loading={loading} isOpen className={styleOverridesForStickyNavigation}>
```

Comment:
idk why, but for some reason removing the collapse from here breaks the crossfade animation:

before (kinda, it's this branch but i just put the Collapse back here):

https://user-images.githubusercontent.com/1170767/187899814-ecb6b8e5-cd1d-4ea8-8c17-b8dc10da1c35.mov

this branch:

https://user-images.githubusercontent.com/1170767/187900266-cca7b191-392f-429b-97e0-4d75ce57a5fe.mov

Reply:
(slightly updated the layout, i've described it in more detail below)